src/main/java/aztech/modern_industrialization/nuclear/FluidNuclearComponents.java

```diff
@@ -51,49 +56,51 @@ public class FluidNuclearComponents {
     public static INuclearComponent<FluidVariant> of(FluidVariant variant) {
         Fluid fluid = variant.getFluid();
 
-        if (fluid == Fluids.WATER) {
-            return WATER;
-        } else if (fluid == MIFluids.HEAVY_WATER.asFluid()) {
-            return HEAVY_WATER;
-        } else if (fluid == MIFluids.HIGH_PRESSURE_WATER.asFluid()) {
-            return HIGH_PRESSURE_WATER;
-        } else if (fluid == MIFluids.HIGH_PRESSURE_HEAVY_WATER.asFluid()) {
-            return HIGH_PRESSURE_HEAVY_WATER;
```

Comment:
This whole function can be a `get` call at this point. :P

Reply:
The _of_ function?

Reply:
yeah - `get` will return null if the value is not in the map already

Reply:
You're right xD
Fixed it